util/dHomogTrans.m

```diff
@@ -1,4 +1,4 @@
-function ret = dHomogTrans(T, S, qdotToV)
+function ret = dHomogTrans(T, S, qdot_to_v)
```

Comment:
This turned out to be way more efficient. Copied over from C++ after figuring out a nicer way.